src/status_im2/contexts/onboarding/identifiers/profile_card/view.cljs

```diff
@@ -0,0 +1,71 @@
+(ns status-im2.contexts.onboarding.identifiers.profile-card.view
+  (:require [react-native.core :as rn]
+            [react-native.reanimated :as reanimated]
+            [react-native.masked-view :as masked-view]
+            [reagent.core :as reagent]
+            [quo2.core :as quo]
+            [quo2.foundations.colors :as colors]
+            [utils.worklets.identifiers-highlighting :as worklets.identifiers-highlighting]
+            [status-im2.contexts.onboarding.identifiers.profile-card.style :as style]))
+
+(defn- f-profile-card-component
+  [{:keys [profile-picture name hash emoji-hash
+           customization-color progress]
+    :or   {customization-color :turquoise}}]
+  (let [container-background (worklets.identifiers-highlighting/background
+                              (colors/custom-color customization-color 50)
+                              @progress)
+        opacity              (worklets.identifiers-highlighting/opacity @progress)
+        avatar-opacity       (worklets.identifiers-highlighting/avatar-opacity @progress)
+        ring-opacity         (worklets.identifiers-highlighting/ring-opacity @progress)
+        user-hash-color      (worklets.identifiers-highlighting/user-hash-color @progress)
+        user-hash-opacity    (worklets.identifiers-highlighting/user-hash-opacity @progress)
+        emoji-hash-style     (worklets.identifiers-highlighting/emoji-hash-style @progress)]
+
+    [rn/view
+     {:key   (str name)
+      :style style/card-view
+      :holes []}
+     [reanimated/view
+      {:style (style/card-container container-background)}
+      [rn/view {:style style/card-header}
+       [reanimated/view
+        {:style (reanimated/apply-animations-to-style
+                 {:opacity avatar-opacity}
+                 {})}
```

Comment:
I'm not sure about what did we agree about  animated styles. Whether they should be in the `style` namespace or in the `view` one.

For this PR, I suggest to be consistent, because some of them are in the `style` ns and this one is in `view`.